apps/meteor/client/views/marketplace/AppsProvider.tsx

```diff
@@ -351,6 +412,17 @@ const AppsProvider: FC = ({ children }) => {
 					reload: fetch,
 				});
 
+				if (app.private) {
+					dispatchPrivateApps({
+						type: 'update',
+						app: {
+							...app,
+							status,
+						},
+						reload: fetch,
+					});
```

Comment:
No `return`? It's supposed to dispatch twice?

Reply:
two different lists